File: src/test/console/php/services/DeviceService.php
<?php

function getOnlineDevice($platform) {
  include("configs.php");

  $url = $testServerUrl.'/devices/bookable/'.$platform.'/1';
  $ch = curl_init($url);

  $headers = array(
    "token: ".$testServerSecretKey
  );
  curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
  curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
  $body = curl_exec($ch);

  curl_close($ch);
  $allDevices = json_decode($body);

  $device = $allDevices[0];
  return $device;
}

File: src/test/console/python/services/DeviceService.py
import requests
import sys
sys.path.append('../..')
from python import configs

class DeviceService:
  def getOnlineDevice(self, platformName):
    hostUrl = configs.testServerUrl
    headers = {'token': configs.testServerSecretKey}
    url = '{}/devices/bookable/{}/1'.format(hostUrl, platformName)
    rq = requests.get(url, headers = headers)

    if rq.status_code == 200:
      device = rq.json()[0]
    else:
      device = None

    return device
